### PR TITLE
tests are ongoing

### PR DESCRIPTION
**Pull request by karinka83**
tests are ongoing

### CODE REVIEW

File: src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+        /* 1. Открыть базовую страницу
+        2. Нажать на пункт меню BROWSE LANGUAGES
+        3. Нажать на подменю J
+        4. Подтвердить, что пользователь видит текст “All languages starting with the letter J are shown, sorted by Language.”
+        */
+
+    @Test
+    public void test_TC_12_01() {
+        getDriver().get(baseURL);
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java:23]: You might add space after this line.

File: src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+public class Song99BottlesKarinaKTest extends BaseTest {
+
+    public static final String baseURL = "http://www.99-bottles-of-beer.net/";
+
+        /* 1. Открыть базовую страницу
+        2. Нажать на пункт меню BROWSE LANGUAGES
+        3. Нажать на подменю J
+        4. Подтвердить, что пользователь видит текст “All languages starting with the letter J are shown, sorted by Language.”
+        */
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java:19]: Comments are better to be deleted.

File: src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        String expectedResult = getDriver().findElement(By.xpath("//tbody/tr[22]/td[2]")).getText();
+
+        getDriver().get(baseURL);
+        getDriver().findElement(By.xpath("//li/a[@href='/abc.html']")).click();
+        String actualResult = getDriver().findElement(By.xpath("//tbody/tr[22]/td[1]/a")).getText();
+        //WebElement table = getDriver().findElement(By.xpath("//table[@id='category']"));
+
+        Assert.assertEquals(actualResult, expectedResult);
+
+     */
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java:60]: Comments are better to be deleted.

**Review by anonymoustuna — APPROVED** [review]

**Review by eglya — APPROVED** [review]
I read in slack the teacher of the beginner's group wrote: comments are required and your group has another standard of "beautiful" code. That's why it LGTM.

File: src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java (hunk 1 above)
Comment (karinka83) [reply src/test/java/qa_java_beginners/Song99BottlesKarinaKTest.java:23]: Thank you for advice

**Pull request merged**